gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.script.lang.kotlin.accessors.auto=true
```

Comment:
This allowed me to not have all that extra fully classified cruft at the bottom of the `build.gradle.kts` file. Thanks @nightpool for suggesting that I clean that up or I wouldn't have tried this new config mode.